src/simulator/non_physical_object.py

```diff
@@ -0,0 +1,8 @@
+import object
+
+
+class Non_physical_object(object.Object):
+    """
+    Object without physical existence, it does not intersect with objects
+    """
+    pass
```

Comment:
now that I'm thinking, Beam might actually have some code. This will depend on the sensor. So maybe we can have Anahita begin her "fixed sensor" task and then get back to this :-?

Reply:
I agree with you, should we leave it as it is for now, or change it?

Reply:
Well I can't think of anything right now. Let's leave it as it is.